extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -163,9 +143,8 @@ private function prepareWikiVariables() {
 			);
 		}
 
-		$wikiVariables['navigation2016'] = $navigation;
+		$wikiVariables['localNav'] = $navigation;
```

Comment:
This change isn't backwards compatible. 
Please remember that we release app first. Do you have a plan how to avoid problems because of that?
Quickly I found two options:
1. Release mercury only changes and then merge app changes
2. Split app changes into 2 phases.